spec/system/articles/user_visits_articles_by_timeframe_spec.rb

```diff
@@ -30,7 +30,7 @@ def shows_main_article
       it "shows correct articles", :aggregate_failures do
         shows_correct_articles_count(2)
         shows_main_article
-        within("#articles-list") do
+        within("#main-content") do
```

Comment:
test (blocking): Add an e2e test for the main page only (for now until #1153 is implemented) to ensure the skip link appears and has focus. When enter or click occurs on the skip link also ensure it brings focus to the main context.

We can also leave this system test for now as it is testing something else.

Reply:
This is a bit more tricky than I first thought - we can't simulate tab events in Cypress (https://docs.cypress.io/api/commands/type.html#Events) which makes this hard to test in a meaningful way. Focus isn't transferred to the `<main>` on skip link click, it effectively just repositions where the next `tab` will focus (in this case should be "Feed" link inside main). Without simulating the tab event I can't validate that's happening.

Similarly without the tab event, I can't validate that the skip link is focused on first `tab`, and have to manually call `focus()` to make it visible.

I could add a test that checks for the skip link being there, clickable, and with the expected href  but it doesn't feel like what we want to me. What do you think? That cypress link above ^ suggests a library that can give us tab-ability but the line `this module is in beta, and might cause some strange failures.` gives me pause

Reply:
Thanks for looking into this. I didn't realize what I was suggesting was currently not possible.

In that case it doesn't make sense for now.

Reply:
me either - I assumed this was definitely possible

Reply:
Just linking to the Cypress issue about enabling tabbing as part of their native events work, https://github.com/cypress-io/cypress/issues/311